### ISSUE TITLE
240FPS plays at normal speed

### ISSUE THREAD

**Issue by XEonAX** [XEonAX/ANXCamera10#6]
Video is actually recorded at 240fps. Currently in the absence of MiuiGallery you need to use a video player/editor capable of slowing the video down to 24fps. Try Google Photos, or MXPlayer.

**Comment by Aurnytoraink** [comment]
I don't think this is a bug, I mean like that can control how they want their slow motion
